### PR TITLE
Eda

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR performs EDA on the data

### CODE REVIEW

File: Wave1_data/5.exploratory_data_analysis/run_all_eda_locally.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# activate the conda environment
+conda activate pyroptosis_timnelapse_env
+
+# convert the notebooks to scripts
+jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/ notebooks/*.ipynb
+cd scripts || exit
+
+# run the scripts
+# python 0.preprocess_profiles.py
+# python 1.eda_compute.py
```
Comment (jaceybronte) [inline Wave1_data/5.exploratory_data_analysis/run_all_eda_locally.sh]: Why is this commented out?

File: Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
+import pathlib
+from pprint import pprint
+
+import pandas as pd
+import pyarrow.parquet as pq
+
+
+# In[ ]:
```
Comment (jaceybronte) [inline Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py:15]: Looks like this wasn't run all the way through

File: Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        )
+        # save the subset data
+        data.to_parquet(subset_data_output_file_path)
+        data.head()
+    else:
+        data = pd.read_parquet(input_data_dict[dataset]["input_file_path"])
+        data.head()
+
+    # perform preprocessing on each data
+    # sort the time and replace with 1, 2, 3, 4
```
Comment (jaceybronte) [inline Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py:82]: Why is this done?

File: Wave1_data/5.exploratory_data_analysis/scripts/1.eda_compute.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pyarrow.parquet as pq
+import seaborn as sns
+import umap
+from sklearn.decomposition import PCA
+
+
+# In[ ]:
```
Comment (jaceybronte) [inline Wave1_data/5.exploratory_data_analysis/scripts/1.eda_compute.py:20]: Also doesn't seem to be run all the way through

**Review by jaceybronte — APPROVED** [review]
Couple of small things, but otherwise LGTM!

File: Wave1_data/5.exploratory_data_analysis/run_all_eda_locally.sh (hunk 1 above)
Comment (MikeLippincott) [reply Wave1_data/5.exploratory_data_analysis/run_all_eda_locally.sh]: for just visualizing and not running. Uncommented the code

File: Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py (hunk 3 above)
Comment (MikeLippincott) [reply Wave1_data/5.exploratory_data_analysis/scripts/0.preprocess_profiles.py:82]: The plate metadata is the time metadata essentially in this data

**Pull request merged**